v3/qwertykeys/qk100/qk100_solder.json

```diff
@@ -0,0 +1,368 @@
+{
+  "name": "qk100 Solder",
+  "vendorId": "0x4F53",
+  "productId": "0x3053",
+  "keycodes": ["qmk_lighting"],
+  "menus": [
+    {
+      "label":"Lighting",
+      "content": [
+        {
+          "label": "LED Strip",
+          "content": [
+            {
+              "label": "Brightness",
+              "type": "range",
+              "options": [0, 120],
+              "content": ["id_qmk_rgblight_brightness", 2, 1]
+            },
+            {
+              "label": "Effect",
+              "type": "dropdown",
+              "content": ["id_qmk_rgblight_effect", 2, 2],
+              "options": [
+                "All Off",
+                "Solid Color",
+                "Breathing 1",
+                "Breathing 2",
+                "Breathing 3",
+                "Breathing 4",
+                "Rainbow Mood 1",
+                "Rainbow Mood 2",
+                "Rainbow Mood 3",
+                "Rainbow Swirl 1",
+                "Rainbow Swirl 2",
+                "Rainbow Swirl 3",
+                "Rainbow Swirl 4",
+                "Rainbow Swirl 5",
+                "Rainbow Swirl 6",
+                "Snake 1",
+                "Snake 2",
+                "Snake 3",
+                "Snake 4",
+                "Snake 5",
+                "Snake 6",
+                "Knight 1",
+                "Knight 2",
+                "Knight 3",
+                "Christmas",
+                "Gradient 1",
+                "Gradient 2",
+                "Gradient 3",
+                "Gradient 4",
+                "Gradient 5",
+                "Gradient 6",
+                "Gradient 7",
+                "Gradient 8",
+                "Gradient 9",
+                "Gradient 10",
+                "RGB Test",
+                "Alternating",
+                "Twinkle 1",
+                "Twinkle 2",
+                "Twinkle 3",
+                "Twinkle 4",
+                "Twinkle 5",
+                "Twinkle 6"
+              ]
+            },
+            {
+              "showIf": "{id_qmk_rgblight_effect} != 0",
+              "label": "Effect Speed",
+              "type": "range",
+              "options": [0, 255],
+              "content": ["id_qmk_rgblight_effect_speed", 2, 3]
+            },
+            {
+              "showIf": "{id_qmk_rgblight_effect} != 0 && {id_qmk_rgblight_effect} != 35",
+              "label": "Color",
+              "type": "color",
+              "content": ["id_qmk_rgblight_color", 2, 4]
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "matrix": { 
+    "rows": 6, 
+    "cols": 19
+  },
+  "layouts": {
+    "labels": [
+      "2U Home",
+      "Split Backspace",
+      "ISO Enter",
+      "Split Left Shift"
+    ],
+    "keymap": [
+      [
```

Comment:
![image](https://github.com/the-via/keyboards/assets/17491233/fd6df849-cd2d-4699-9672-8122399115af)

suggest moving ISO out a bit (both files)
![image](https://github.com/the-via/keyboards/assets/17491233/12020b06-226e-4e45-9d94-bb4ed98b3697)